File: Python/1113 - Ascending and Descending.py
stop = False
while (stop != True):
    s = input().split()
    for i in range(len(s)):
        s[i] = int(s[i])
    if (s[0] == s[1]):
        stop = True
    else:
        if(s[0] > s[1]):
            print("Decrescente")
        else:
            print("Crescente")

File: C++/1113 - Ascending and Descending.cpp
#include <iostream>
 
int main() {
    int x, y;
    do
    {
        std::cin >> x >> y;
        if (x == y)
        {
            break;
        }
        if (x > y)
        {
            std::cout << "Decrescente\n";
        }
        else
        {
            std::cout << "Crescente\n";
        }
    }
    while (x != y);
    return 0;
}